<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="charger">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Charger Graph</string>
     </property>
    </widget>
    <widget class="QPushButton" name="afficherNodes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher Nodes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="etapeSuivante">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Etape Suivante</string>
     </property>
    </widget>
    <widget class="QPushButton" name="graphFinale">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Graph Finale</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="zoomEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>41</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QSpinBox" name="widthTopo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="heigthTopo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="widthCar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="heigthCar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Generer Fichier</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="nbrNodes">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="rangeEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="densite">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>charger</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>chargerGraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>afficherNodes</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>afficherNodes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphFinale</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>afficherGraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearGraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>genererTopologie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chargerGraph()</slot>
  <slot>afficherNodes()</slot>
  <slot>afficherGraph()</slot>
  <slot>clearGraph()</slot>
  <slot>genererTopologie()</slot>
 </slots>
</ui>
